src/fsearch_window_actions.c

```diff
@@ -321,6 +321,20 @@ open_with_cb (GtkTreeModel *model,
     }
 }
 
+void
+fsearch_window_action_after_file_open(bool action_mouse) {
+    FsearchConfig *config = fsearch_application_get_config (FSEARCH_APPLICATION_DEFAULT);
+
+    if ((config->action_after_file_open_keyboard && !action_mouse)
+    || (config->action_after_file_open_mouse && action_mouse)) {
+        if (config->action_after_file_open == 1) {
+            g_application_quit (G_APPLICATION (FSEARCH_APPLICATION_DEFAULT));
+        } else {
+            gtk_window_iconify (gtk_application_get_active_window (G_APPLICATION (FSEARCH_APPLICATION_DEFAULT)));
```

Comment:
`gtk_application_get_active_window` expects a `GtkApplication *` so `G_APPLICATION` should rather be `GTK_APPLICATION`.